tests/ansible_collections/roles/oracle-linux-specific/tasks/boot_standard_kernel.yml

```diff
@@ -10,7 +10,3 @@
 
 - name: Set default kernel to Red Hat compatible kernel
   shell: "grubby --set-default /boot/vmlinuz-{{ kernel_ver.stdout }}"
-
-- name: Reboot
```

Comment:
How is this reboot replaced?

Reply:
The reboot is done for everything after the "main preparation step" in `plans/main.fmf` file.